components/omega/doc/devGuide/DataTypesDev.md

```diff
@@ -0,0 +1,52 @@
+(omega-dev-data-types)=
```

Comment:
I would probably just name the file `DataTypes.md` (and the same in the User's Guide) to keep things from getting too redundant.  But let me know if there's a reason you think the `Dev` and `User` suffixes are useful.

Reply:
This file should be added to the main `index.md` just so it shows up in the documentation.  I realize that things are going to be pretty disorganized as we get started but I still think it's better if everything we add at least shows up in the docs, and gets organized better in the future.

Reply:
Something like this:
````
```{toctree}
:caption: User's guide
:maxdepth: 2

userGuide/QuickStart
userGuide/DataTypes
```

```{toctree}
:caption: Developer's guide
:maxdepth: 2

devGuide/QuickStart
devGuide/DataTypes
devGuide/CondaEnv
devGuide/Docs
devGuide/BuildDocs
```
````

Reply:
Yeah, it was mostly just nervousness over having files of the same name running around, but thinking about it, that's going to apply to just about everything with a footprint in both user and dev docs so I'll change the names and update the index.